test/views/declaration/TaricCodeAddFirstViewSpec.scala

```diff
@@ -98,7 +102,7 @@ class TaricCodeAddFirstViewSpec extends UnitViewSpec with ExportsTestData with S
       view must haveGovukGlobalErrorSummary
       view must containErrorElementWithTagAndHref("a", "#taricCode")
 
-      view must containErrorElementWithMessageKey("declaration.taricAdditionalCodes.error.invalid")
+      view must containErrorElementWithMessageKey(s"$prefix.error.invalid")
```

Comment:
Nit: Making the i18n key dynamically generated with `$prefix` in the test just makes it more difficult to find uses of this key in the IDE for future refactoring work. I know it is DRYer, but in this instance I think it is counter productive (IMO :slightly_smiling_face: )

Reply:
:+1: